### PR TITLE
First pass at ejb

### PR DESCRIPTION
**Pull request by ggam**

### CODE REVIEW

File: src/main/jbake/content/ejb-intro004.adoc [hunk 1]
```diff
@@ -70,7 +70,7 @@ and Directory Interface syntax to find the enterprise bean instance.
 
 Dependency injection is the simplest way of obtaining an enterprise bean
 reference. Clients that run within a Jakarta EE server-managed environment,
-JavaServer Faces web applications, JAX-RS web services, other enterprise
+Jakarta Server Faces web applications, Jakarta RESTful web services, other enterprise
```
Comment (ggam) [inline src/main/jbake/content/ejb-intro004.adoc]: I didn't capitalize "Web Services" on purpose. Hopefully we'll be able to use "Jakarta REST" as short name, but I'm still not sure about it.

File: src/main/jbake/content/ejb-basicexamples005.adoc [hunk 2]
```diff
@@ -304,7 +304,7 @@ Timer timer = timerService.createCalendarTimer(schedule);
 
 For details on the method signatures, see the `TimerService` API
 documentation at
-`http://docs.oracle.com/javaee/7/api/javax/ejb/TimerService.html`.
+`https://javaee.github.io/javaee-spec/javadocs/javax/ejb/TimerService.html`.
```
Comment (bshannon) [inline src/main/jbake/content/ejb-basicexamples005.adoc]: Let me see if I can guess this correctly...

https://jakarta.ee/specifications/platform/8/apidocs/javax/ejb/TimerService.html

File: src/main/jbake/content/ejb-intro008.adoc [hunk 3]
```diff
@@ -11,7 +11,7 @@ prev=ejb-intro007.html
 Further Information about Enterprise Beans
 ------------------------------------------
 
-For more information on Enterprise JavaBeans technology, see the Enterprise JavaBeans 3.2 specification:
+For more information on Jakarta Enterprise Beans technology, see the Enterprise JavaBeans 3.2 specification:
```
Comment (bshannon) [inline src/main/jbake/content/ejb-intro008.adoc]: "Enterprise JavaBeans" -> "Jakarta Enterprise Beans"

File: src/main/jbake/content/ejb-intro008.adoc [hunk 4]
```diff
@@ -11,7 +11,7 @@ prev=ejb-intro007.html
...
 +
 `http://www.jcp.org/en/jsr/detail?id=345`
```
Comment (bshannon) [inline src/main/jbake/content/ejb-intro008.adoc]: https://jakarta.ee/specifications/enterprise-beans/3.2/

File: src/main/jbake/content/ejb-basicexamples005.adoc (hunk 2 above)
Comment (ggam) [reply src/main/jbake/content/ejb-basicexamples005.adoc]: Indeed you did!

**Comment by ggam** [comment]
> Server Faces, underlines, you know the drill. :-)
> And a few other minor comments below.

I left Server Faces until Arjan confirms us his plans to rename the spec. Otherwise, all done.

**Pull request merged**